docs/releasing/make-helm-chart.sh

```diff
@@ -31,4 +31,6 @@ gsed -i "s/^version:.*/version: ${TAG:1}/" Chart.yaml
 helm package .
 helm push onlineboutique-${TAG:1}.tgz oci://$HELM_CHART_REPO
 
+rm onlineboutique-${TAG:1}.tgz
```

Comment:
**Minor fix:**
```suggestion
rm ./helm-chart/onlineboutique-${TAG:1}.tgz
```
since we run the `make-release.sh` script from the repo root. [Related docs](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/docs/releasing#creating-a-new-release).

Reply:
Tested locally. ✅